webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/apidata/JsonResponseObjects.scala

```diff
@@ -2117,10 +2092,7 @@ trait RudderJsonEncoders {
 
   implicit val vmTypeEncoder: JsonEncoder[VmType] = JsonEncoder[String].contramap(_.name)
 
-  implicit val agentTypeEncoder:         JsonEncoder[AgentType]           = JsonEncoder[String].contramap(_.displayName)
-  implicit val agentVersionEncoder:      JsonEncoder[AgentVersion]        = JsonEncoder[String].contramap(_.value)
   implicit val nodeKindEncoder:          JsonEncoder[NodeKind]            = JsonEncoder[String].contramap(_.name)
-  implicit val securityTokenEncoder:     JsonEncoder[SecurityToken]       = JsonEncoder[String].contramap(_.key)
```

Comment:
these two are moved along in the corresponding companion objects of their classes